apiserver/debuglog_db.go

```diff
@@ -73,19 +73,19 @@ func handleDebugLogDBRequest(
 
 func makeLogTailerParams(reqParams debugLogParams) corelogger.LogTailerParams {
 	tailerParams := corelogger.LogTailerParams{
-		MinLevel:      reqParams.filterLevel,
-		NoTail:        reqParams.noTail,
-		StartTime:     reqParams.startTime,
-		InitialLines:  int(reqParams.backlog),
```

Comment:
Hi Ian, do you think InitialLogCount would be more clear? I'm trying to avoid lines because a log can have multiple lines depending on the size of the terminal